src/docs/device-apis.md

```diff
@@ -15,11 +15,17 @@ For non-crna projects created with vue-native, you may use packages such as `rea
 Vue-native projects, need to import the `sensors` from `expo`.
 
 ```shell
-npm i expo
+npm i expo-accelerometer
+
+```
+or yarn
```

Comment:
We avoid keeping yarn commands in the docs now, since the commands are redundant and it is fairly easy to get the equivalent commands if the npm command is known.